watson/cli.py

```diff
@@ -452,46 +457,62 @@ def report(watson, current, from_, to, projects,
 
     if format_json:
         click.echo(json.dumps(report, indent=4, sort_keys=True))
+        return
+
+    lines = []
+    # use the pager, or print directly to the terminal
+    if pager or (pager is None and
+            watson.config.getboolean('options', 'pager')):
```

Comment:
Should be `watson.config.getboolean('options', 'pager', True)`, no?

Reply:
Also, I'd line up this line with the second `pager` above.

Reply:
Adding 'True' maintains the current default (at least for log) of running output through the pager.

You had me remove this earlier because it was bumping it onto a second line, but the indentation is different now, so that's no longer an issue.

Both fixed!

Reply:
I was trying to say, that you don't need the argument name, i.e `default=`, not the argument value itself.

Reply:
Aaah, that makes more sense. Updated as recommended!